src/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\EventListener;
+
+
+use App\Exception\ResourceValidationException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+
+class ExceptionListener
+{
+    public function onKernelException(ExceptionEvent $event)
```

Comment:
Could be avoided if you use the fos exception controller:  https://symfony.com/doc/master/bundles/FOSRestBundle/4-exception-controller-support.html

Reply:
Done using [these configuration settings](https://openclassrooms.com/fr/courses/4087036-construisez-une-api-rest-avec-symfony/4333906-gestion-des-erreurs#/id/r-4336864)